### PR TITLE
Add the section of learning rate schedulers to docs

### LINKED ISSUE #500: Add schedulers section to bolts docs
https://pytorch-lightning-bolts.readthedocs.io/en/latest/api/pl_bolts.optimizers.lr_scheduler.html

This should be a section in the docs

### PR DESCRIPTION
**Pull request by akihironitta**
## What does this PR do?
Closes #500.

## Before submitting
- [x] Was this discussed/approved via a Github issue? (no need for typos and docs improvements)
- [x] Did you read the [contributor guideline](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/.github/CONTRIBUTING.md), Pull Request section?
- [x] Did you make sure your PR does only one thing, instead of bundling different changes together?
- [x] Did you make sure to update the documentation with your changes?
- [x] Did you write any new necessary tests? [not needed for typos/docs]
- [x] Did you verify new and existing tests pass locally with your changes?
- [x] If you made a notable change (that affects users), did you update the [CHANGELOG](https://github.com/PyTorchLightning/pytorch-lightning-bolts/blob/master/CHANGELOG.md)?

## PR review
 - [x] Is this pull request ready for review? (if not, please submit in draft mode)

Anyone in the community is free to review the PR once the tests have passed.
If we didn't discuss your PR in Github issues there's a high chance it will not be merged.

## Did you have fun?
Make sure you had fun coding 🙃

### CODE REVIEW

File: docs/source/learning_rate_schedulers.rst [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Learning Rate Schedulers
+========================
+This package lists common losses across research domains
+(This is a work in progress. If you have any losses you want to contribute, please submit a PR!)
```
Comment (ydcjeff) [inline docs/source/learning_rate_schedulers.rst:4]: Its scheduler, right?
Not losses

**Review by Borda — APPROVED** [review]
